tox.ini

```diff
@@ -30,12 +30,8 @@ whitelist_externals = rm
 [testenv:lint]
 deps =
      {[base]testdeps}
-     pycodestyle
-     pydocstyle
      pylint
 commands =
-    pycodestyle {[base]name} --exclude tests
-    pydocstyle {[base]name}
```

Comment:
Note that these checks will no more be executed in CI until we setup pre-commit.ci.

Reply:
I'd like to keep the normal path of checking these things, so that we don't change our workflow too much as we try precommit - can they coexist for the time being?

Reply:
Sure they can. It will be executed twice when we will setup pre-commit.ci but it is no big deal.